gatsby-config.js

```diff
@@ -27,6 +28,83 @@ module.exports = {
                 includeInDevelopment: false,
             },
         },
+        {
+            resolve: `gatsby-plugin-feed`,
+            options: {
+                query: `
+                {
+                    site {
+                    siteMetadata {
+                        title
+                        siteUrl
+                    }
+                    }
+                }
+                `,
+                feeds: [
+                    {
+                        serialize: ({ query: { site, allSitePage } }) => {
+                            return allSitePage.edges.map(edge => {
+                                const path = dlv(edge, 'node.path');
+                                const query_fields = dlv(
+                                    edge,
+                                    'node.context._xrefDocMapping.query_fields'
+                                );
+                                const description = dlv(
+                                    query_fields,
+                                    [
+                                        'meta_description',
+                                        0,
+                                        'children',
+                                        0,
+                                        'value',
```

Comment:
Wes should use get-nested-value for this in case there are any intermediate nodes.